mungegithub/README.md

```diff
@@ -39,7 +39,8 @@ TARGET=kubernetes REPO=docker.io/$USERNAME APP=submit-queue KUBECONFIG=/path/to/
 ```
 
 **Note** The submit-queue should always be deployed from master so that look at
-the master branch (configmap, etc) reflects what is currently running.
+the master branch (configmap, etc) reflects what is currently running. The deployment may require
+checkout based on commit hash found in [Godeps.json](https://github.com/kubernetes/test-infra/blob/master/Godeps/Godeps.json).
```

Comment:
This isn't super clear to me.  I should go into the godeps file, find the appropriate hash and checkout all of the appropriate commits from everyone of the repos.